### PR TITLE
allow bumping version in multiple files by passing them in

### PR DESCRIPTION
**Pull request by grosser**
this will be useful to update version in the Readme or other static places

will use this to replace the hacky parts of my `automatically release a ruby binary` script :)
https://github.com/grosser/git-autobisect/commit/1850359b60f4119a2e2a27797fac4e7659ddcfdc

@gregorym

### CODE REVIEW

File: lib/bump.rb [hunk 1]
```diff
@@ -82,13 +83,26 @@ def parse_cli_options_value(value)
...
         if options[:bundle] and Dir.glob('*.gemspec').any? and under_version_control?("Gemfile.lock")
           bundler_with_clean_env do
             return ["Bundle error", 1] unless system("bundle")
+            git_add "Gemfile.lock", options
           end
         end
-        commit(next_version, file, options) if options[:commit]
+
+        # commit staged changes
+        commit(next_version, options)
```
Comment (grosser) [inline lib/bump.rb]: separating staging from committing so things are more pluggable

File: lib/bump.rb [hunk 2]
```diff
@@ -115,16 +129,24 @@ def commit_message(version, options)
...
+        system("git commit -m '#{commit_message(version, options)}'")
         system("git tag -a -m 'Bump to v#{version}' v#{version}") if options[:tag]
       end
 
+      def git_add(file, options)
+        system("git add --update #{file}") if commit_requested?(options)
+      end
+
+      def commit_requested?(options)
+        options[:commit] && File.directory?(".git")
```
Comment (grosser) [inline lib/bump.rb]: ... if commit was requested and git is not there that should blow up too, but for now keeping the old behavior ...

File: README.md [hunk 3]
```diff
@@ -49,6 +49,12 @@ If you don't want to run the `bundle` command after bumping, add the `--no-bundl
 
     bump patch --no-bundle
 
+### `--reaplace-in`
```
Comment (gregorym) [inline README.md]: Do you mean `--replace-in`? 😃

**Comment by grosser** [comment]
also made all the `system` calls safer by using arrays, so no more funky bugs when using quotes in messages :)

**Comment by grosser** [comment]
all good now, please release after this ... minor version I'd say

**Pull request merged**